flex/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
Why do you use 3.20.0 here, if you propose 3.21.0 for the parent project and bison?